### PR TITLE
Added Seldon Issue NOTE to Industrial Edge 

### PR DESCRIPTION
**Pull request by claudiol**

### CODE REVIEW

File: content/patterns/industrial-edge/_index.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ aliases: /industrial-edge/
   </script>
 </div>
 
+---
+**NOTE**
+
+Industrial Edge on OpenShift 4.12 fails CI due to Seldon issue. This only affects the
```
Comment (abhatt-rh) [inline content/patterns/industrial-edge/_index.md:32]:
```suggestion
Industrial Edge on OpenShift Container Platform 4.12 fails CI due to Seldon issue. This only affects the 
```

File: content/patterns/industrial-edge/_index.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ aliases: /industrial-edge/
   </script>
 </div>
 
+---
+**NOTE**
+
+Industrial Edge on OpenShift 4.12 fails CI due to Seldon issue. This only affects the 
+Anomaly Detection AI/ML portion of the pattern.  The rest of the pattern functions as
+designed.  For more information on the Seldon issue please refer to https://github.com/SeldonIO/seldon-core/issues/4339.
```
Comment (abhatt-rh) [inline content/patterns/industrial-edge/_index.md:34]:
```suggestion
designed. For more information on the Seldon issue, see https://github.com/SeldonIO/seldon-core/issues/4339.
```

File: content/patterns/industrial-edge/_index.md [hunk 3]
```diff
@@ -26,6 +26,15 @@ aliases: /industrial-edge/
   </script>
 </div>
 
+---
+**NOTE**
+
+Industrial Edge on OpenShift 4.12 fails CI due to Seldon issue. This only affects the 
+Anomaly Detection AI/ML portion of the pattern.  The rest of the pattern functions as
```
Comment (abhatt-rh) [inline content/patterns/industrial-edge/_index.md:33]:
```suggestion
Anomaly Detection AI/ML portion of the pattern. The rest of the pattern functions as
```

File: content/patterns/industrial-edge/_index.md
Comment (abhatt-rh) [inline content/patterns/industrial-edge/_index.md:1]: Remove the hard line-breaks

**Review by abhatt-rh — COMMENTED** [review]
Hi @claudiol, just added a few nits, else LGTM!

File: content/patterns/industrial-edge/_index.md (hunk 1 above)
Comment (claudiol) [reply content/patterns/industrial-edge/_index.md:32]: Thank you @abhatt-rh !

**Pull request merged**